documents/documents_test/service_test.go

```diff
@@ -340,9 +419,18 @@ func TestService_Exists(t *testing.T) {
 
 }
 
-func createCDWithEmbeddedInvoice(t *testing.T, ctx context.Context, skipSave bool) (documents.Model, coredocumentpb.CoreDocument) {
+func createCDWithEmbeddedInvoice(t *testing.T, ctx context.Context, collaborators []identity.DID, skipSave bool) (documents.Model, coredocumentpb.CoreDocument) {
```

Comment:
maybe this function could be moved to the testingdocuments package

Reply:
That would create a dependency cycle AFAIK